pyne/cpp_source.pxd

```diff
@@ -0,0 +1,34 @@
+from libcpp.map cimport map as cpp_map
+from libcpp.string cimport string as cstr
+from libcpp.vector cimport vector
+
+cdef extern from "source.h" namespace "pyne":
+
+    cdef cppclass PointSource:
+        # constructors
+        PointSource() except +
+        PointSource(double) except +
+        PointSource(double, double) except +
+        PointSource(double, double, double) except +
+        PointSource(double, double, double, double) except +
+        PointSource(double, double, double, double, double) except +
+        PointSource(double, double, double, double, double, double) except +
+        PointSource(double, double, double, double, double, double, double) except +
+        PointSource(double, double, double, double, double, double, double, cstr) except +
+        PointSource(double, double, double, double, double, double, double, cstr, double) except +
```

Comment:
I know you said docstrings are missing, but also be sure to include what the difference is with all these constructors.

Reply:
we don't usually put docstring in the cpp_XXX.pxd.

those are defined in the `.cpp/.hpp`